Example/App/DotSpinnerButton.js

```diff
@@ -29,8 +29,8 @@ const DotSpinnerButton = () => {
           gradientName={'Night Fade'}
           gradientRadialRadius={10}
           gradientButtonHeight={50}
-          radialRadiusx={"50%"}
-          radialRadiusy={"50%"}
+          radialRadiuSx={"50%"}
+          radialRadiuSy={"50%"}
```

Comment:
`s` should not be capital `X` and `Y` should be

Reply:
These all the Naming conventions are already taken while production I am just giving typescript Support not changing any component or names because changing names will reflect errors to the whole library as of now I am changing these names according to comments.